doc/getting-started/getting-started.rst

```diff
@@ -567,6 +567,7 @@ Build ACRN
 
       .. code-block:: bash
 
+         cd ~/acrn-work/acrn-kernel
```

Comment:
Would you mind also deleting this lines 610/611:

Even though our sample default scenario defines six User VMs, we're only
going to launch one of them, so we'll only need the one launch script.

Reply:
Got it.